episodes/filters-and-thresholding.md

```diff
@@ -713,18 +690,17 @@ code block in that exercise to open the image:
 rectangle, circle and triangle"}
 
 Try some of the other automatic thresholding options provided by the 
-`napari-segment-blobs-and-things-with-membranes` and 
-`napari-simpleitk-image-processing` plugins.
+`napari-skimage` plugin. Set the 'method' to different options in 
+`Layers > Filter > Thresholding > Automated Threshold (napari skimage)`
 
-For example:
+Try:
 
-- `Tools > Segmentation / binarization > Threshold (Triangle method, Zack et al 1977, scikit-image, nsbatwm)`
-- `Tools > Segmentation / binarization > Threshold (Yen et al 1995, scikit-image, nsbatwm)`
-- `Tools > Segmentation / binarization > Threshold (Li et al 1993, scikit-image, nsbatwm)`
-- `Tools > Segmentation / binarization > Threshold Otsu, multiple thresholds (n-SimpleITK)`
+- `mean`
+- `yen`
+- `li`
+- `sauvola`
 
-How do they compare to standard Otsu thresholding?  
-`Tools > Segmentation / binarization > Threshold (Otsu et al 1979, scikit-image, nsbatwm)`
+How do they compare to standard `otsu` thresholding?
```

Comment:
Not sure why we say standard. The plugin has no options for Otsu, so could just change to.  Although there's a few other instances of "standard Otsu" that would also need changing. 
```suggestion
How do they compare to `otsu` thresholding?
```

Reply:
Good point! I'll remove the 'standard'. This was included because the old plugin had multiple types of otsu e.g. `Threshold Otsu, multiple thresholds`.